src/api/export.ts

```diff
@@ -112,7 +112,8 @@ async function _export({
 		const is_html = type === 'text/html';
 
 		if (is_html) {
-			if (pathname !== '/service-worker-index.html') {
+			if (pathname !== '/service-worker-index.html'
+			   && pathname.substr(pathname.length - 5) !== '.html') {
```

Comment:
maybe we don't need the `pathname !== '/service-worker-index.html'` check anymore since this would cover it?

Reply:
Yep, I did the same tidying in #1049 before merging that in.